nodes/sample_host.json

```diff
@@ -40,9 +40,10 @@
       "redirect_domain_names": ["<domain name>", "<domain name>", "<...>"],
       "ruby_version": "2.1.0",
       "ssl_info": {
-        "key": "<ssl key>",
-        "crt": "<ssl crt>"
-        },
+        "enabled": true,
+        "certificate": "The cert file, optional",
```

Comment:
Sorry; I was merely a bit confused as to what I should have to place in the value of "certificate" and "certificate_key". I assumed it had to the abslute path on my local machine.

I was merely thinking out loud how we could improve the wording so it becomes immediate apparent what we are expecting as values there.